pkg/storage/snapshot/snapshot_test.go

```diff
@@ -1716,6 +1791,62 @@ var _ = Describe("Snapshot controlleer", func() {
 				),
 			)
 
+			It("should set paused indication when VM is paused with guest agent", func() {
+				vm := createLockedVM()
+				vmSource.Add(vm)
+
+				vmi := createVMI(vm)
+				// Add both paused and guest agent conditions
+				vmi.Status.Conditions = append(vmi.Status.Conditions, v1.VirtualMachineInstanceCondition{
+					Type:          v1.VirtualMachineInstancePaused,
+					LastProbeTime: metav1.Now(),
+					Status:        corev1.ConditionTrue,
+				})
+				vmi.Status.Conditions = append(vmi.Status.Conditions, v1.VirtualMachineInstanceCondition{
+					Type:          v1.VirtualMachineInstanceAgentConnected,
+					LastProbeTime: metav1.Now(),
+					Status:        corev1.ConditionTrue,
+				})
+				vmiSource.Add(vmi)
+
+				vmSnapshot := createVMSnapshotInProgress()
+
+				vmSnapshotContent := createVMSnapshotContent()
+				vmSnapshotContent.Status = &snapshotv1.VirtualMachineSnapshotContentStatus{
+					ReadyToUse: pointer.P(false),
+				}
+				vmSnapshotContentSource.Add(vmSnapshotContent)
+				addVirtualMachineSnapshot(vmSnapshot)
+
+				expectedIndications := []snapshotv1.Indication{
```

Comment:
not sure we need to keep adding tests which regular indications since we are deprecating them, so dont think we need to keep testing it